### PR TITLE
Added some more misc unsupported characters to translit_data

### PR DESCRIPTION
**Pull request by benjymous**
Fixes for issue #7

### CODE REVIEW

**Review by abhijitvalluri — COMMENTED** [review]
Could you please use spaces instead of tabs in each of those lines? Also, please make sure that all the replacements are aligned in a column. Right now, EUR and TM are not aligned.

**Comment by benjymous** [comment]
Will do - I think the alignment is a artefact of the tabs - they do align in Android studio (with my font settings, anyway!)

**Comment by benjymous** [comment]
Ok, that should be fixed.  

Btw, I tried to run the transliteration unit tests, but just get an error "Caused by: java.util.MissingResourceException: Could not find the bundle com/ibm/icu/impl/data/icudt58b/translit/root" - not sure if there's something wrong with my setup, or is something just misconfigured for the path?

**Comment by sabi0** [comment]
@benjymous 
There is a bug in Android Studio wrt unit tests classpath - see https://issuetracker.google.com/issues/37137712
They should run fine with `gradle check` though.

File: app/src/main/res/raw/translit_data [hunk 1]
```diff
@@ -102,3 +102,28 @@ U+1F3FF
...
 U+200D
+
+# Some other misc characters that the Fitbit charset doesn't include
+£	GBP
+€   EUR
+¥	JPY
+¢	c
+©	(C)
+®	(R)
+™   TM
```
Comment (sabi0) [inline app/src/main/res/raw/translit_data]: Maybe "(tm)" or "(TM)" would have been better?
E.g. "NEX™" => "NEXTM"

**Comment by abhijitvalluri** [comment]
Sorry for the delay in merging this branch.

**Pull request merged**